pennylane/queuing.py

```diff
@@ -105,8 +112,7 @@ def update_info(cls, obj, **kwargs):
         Args:
             obj: the object with metadata to be updated
         """
-        if cls.recording():
-            cls.active_context().update_info(obj, **kwargs)
+        return cls.active_context().update_info(obj, **kwargs) if cls.recording() else None
```

Comment:
How come this method (and some other methods) have been changed to return objects? This seems out of scope here.

Reply:
Maybe an out-of-scope change.  Just a minor cleanup that my IDE was telling me to do.  Exact same behaviour, just a little more compact.

Reply:
Well, previously we always returned `None`. Now, this may return an object that is `not None` depending on what `cls.active_context().update_info(obj, **kwargs)` returns.

Overall, I think it's better to revert these changes. The new version doesn't seem to make too much sense in our logic. What reason did the IDE mention?

Reply:
I think it was mostly complaining about the `get_info` method. Then I just realized all the other methods have basically the same form and changed them too.

I've reverted all the methods other than `get_info`.

Reply:
Thanks. What issue did it raise with `get_info`?

Reply:
This refactor can be applied to `get_info`, because this method has a `return` statement inside and outside the `if`.

Reply:
Exactly.  `get_info` was returning values in both cases.

Reply:
:100: Thanks!